src/main/java/uk/gov/hmcts/darts/event/service/impl/CleanupCurrentFlagEventProcessorImpl.java

```diff
@@ -23,27 +23,25 @@ public List<Integer> processCurrentEvent() {
         List<Integer> processedEventIdLst = new ArrayList<>();
         log.debug("Batch size to process event ids for {}", batchSize);
         List<Integer> eventEntityReturned = eventRepository.getCurrentEventIdsToBeProcessed(Pageable.ofSize(batchSize));
-        log.debug("Event ids being processed{}",  processedEventIdLst.stream().map(Object::toString)
-            .collect(Collectors.joining(",")));
-        log.debug("Number of Event ids to be processed {}",  processedEventIdLst.size());
-
-        List<Integer> eventPrimaryKeysLst = new ArrayList<>();
-        List<Integer> eventIdsLst = new ArrayList<>();
+        if (log.isDebugEnabled()) {
+            log.debug("Event ids being processed {}", eventEntityReturned
+                .stream()
+                .map(Object::toString)
+                .collect(Collectors.joining(",")));
+            log.debug("Number of Event ids to be processed {}", eventEntityReturned.size());
+        }
 
         eventEntityReturned.forEach(event -> {
-            Integer eventIdPrimaryKey = eventRepository.getTheLatestCreatedEventPrimaryKeyForTheEventId(event);
-            log.debug("Current event primary key is {}", eventIdPrimaryKey);
-
-            eventPrimaryKeysLst.add(eventIdPrimaryKey);
-            eventIdsLst.add(event);
-
+            EventRepository.EventIdAndHearingIds eventIdAndHearingIds = eventRepository.getTheLatestCreatedEventPrimaryKeyForTheEventId(event);
+            log.debug("Current event primary key is {}", eventIdAndHearingIds);
```

Comment:
Is this going to print the whole object? Or do you want to print just one of the primary keys?

Reply:
I have updated the log after this one to include more detail and removed this log as they both showed the same information